codespell_lib/_codespell.py

```diff
@@ -345,8 +345,10 @@ def parse_options(args):
                              'you\'d give "*.eps,*.txt" to this option.')
 
     parser.add_argument('-x', '--exclude-file', type=str, metavar='FILE',
-                        help='FILE with lines that should not be checked for '
-                             'errors or changed')
+                        help='specify a file FILE with whole lines to be '
+                             'excluded from being checked for errors or changed. '  # noqa: E501
```

Comment:
I don't think we need the noqa here, we're just joining strings together already so we can wrap it ourselves so it fits.